scripts/workers/code_upload_submission_worker.py

```diff
@@ -119,6 +119,69 @@ def create_job_object(message, environment_image):
     return job
 
 
+def create_single_pod_job_object(message):
+    """Function to create the single pod AWS EKS Job object
+
+    Arguments:
+        message {[dict]} -- Submission message from AWS SQS queue
+
+    Returns:
+        [AWS EKS Job class object] -- AWS EKS Job class object
+    """
+
+    PYTHONUNBUFFERED_ENV = client.V1EnvVar(name="PYTHONUNBUFFERED", value="1")
+    EVALAI_API_SERVER_ENV = client.V1EnvVar(
+        name="EVALAI_API_SERVER", value=EVALAI_API_SERVER
+    )
+    # Used by agent to create submission file by phase_pk and selecting dataset location
+    MESSAGE_BODY_ENV = client.V1EnvVar(name="BODY", value=json.dumps(message))
+    submission_pk = message["submission_pk"]
+    image = message["submitted_image_uri"]
+
+    volume_mount_list = get_volume_mount_list("/dataset")
+    lifecycle_config = client.V1Lifecycle(
+        pre_stop=client.V1Handler(
+            _exec=client.V1ExecAction(
+                command=["/bin/sh", "-c", "sleep 10"]  # Submit File to evalai
+            )
+        )
+    )
+    # Configureate Pod agent container
+    agent_container = client.V1Container(
+        name="agent",
+        image=image,
+        env=[PYTHONUNBUFFERED_ENV, EVALAI_API_SERVER_ENV, MESSAGE_BODY_ENV],
+        resources=client.V1ResourceRequirements(
+            limits={"nvidia.com/gpu": "1"}
+        ),
+        volume_mounts=volume_mount_list,
+        lifecycle=lifecycle_config,
+    )
+
+    volume_list = get_volume_list()
+    # Create and configurate a spec section
+    template = client.V1PodTemplateSpec(
+        metadata=client.V1ObjectMeta(labels={"app": "evaluation"}),
+        spec=client.V1PodSpec(
+            containers=[agent_container],
+            restart_policy="Never",
+            volumes=volume_list,
+        ),
+    )
+    # Create the specification of deployment
+    spec = client.V1JobSpec(backoff_limit=1, template=template)
+    # Instantiate the job object
+    job = client.V1Job(
```

Comment:
We can create a new method to get the `Job` object. Parameters of this method will be `submission_pk` with needed specs